Add Monthly Beatmapping Contests Return news post (#2989)
ppy/osu-wiki: news/2020-01-03-monthly-beatmapping-contests-return.md

---
layout: post
title: "Monthly Beatmapping Contests Return"
date: 2020-01-03 22:00:00 +0000
---

It's time for a new way to flaunt your beatmapping capabilities. Find out what it takes to become osu!'s next Elite Mapper!

![monthly beatmapping contest banner](/wiki/shared/news/banners/monthly-beatmapping-contest.png)

Beatmapping contests have been far and few between throughout the last couple of years, but that's about to change. The Monthly Beatmapping Contest (MBC) is back with a new format! Participants must adapt to different mapping limitations each month and be evaluated by an unconventional grading system to earn some fancy rewards!

## Timeline

The timeline for each month's contest is as follows:
- [**Beatmapping (3 weeks):**](#beatmapping) Contest is announced and beatmappers create their entries
- [**Screening (1 week):**](#screening) Entries are evaluated by experienced beatmappers
- [**Voting (2 weeks):**](#voting) Top 5 entries from screening are published for the osu! community to vote on
- [**Rewards:**](#rewards) Winners are announced and prizes are redeemed

> "But that's more than one month!", you may be thinking. Each month's contest overlaps, so when a contest is announced at the start of a month, you'll receive access to the voting page for the previous month's contest. Halfway through the month you'll also find out the winner from the previous month's contest (and recieve a reminder of the ongoing contest's submission deadline!).

Reviewer: ```suggestion
"But that's more than one month!", you may be thinking. Each month's contest overlaps, so when a contest is announced at the start of a month, you'll receive access to the voting page for the previous month's contest. Halfway through the month, you'll also find out the winner from the previous month's contest (and receive a reminder of the ongoing contest's submission deadline!).
```

## <a id="beatmapping"></a>Beatmapping

### <a id="limitation"></a>Limitation

Creating a high quality beatmap is a great goal to strive for, but let's be honest, *that makes for a boring competition.* 

The Monthly Beatmapping Contest requires beatmappers to adhere to a mapping limitation each month. This limitation can be anything from a specific circle size to a limit on star rating.

> This month's limitation: **all slider shapes must be straight lines.** No curved sliders or sliders using red anchors are allowed!

Reviewer 2: I think adding an image of examples of allowed/disallowed sliders would work well here to clearly illustrate the requirement visually. Also, are the maps required to use sliders or could someone go circles only? Circles only would technically fit the "no curved/angled sliders" requirement if there are no sliders.

Reviewer 3: means what it says, I don't see any room for confusion here

 With less wiggle room in the editor, we hope to see engaging ways of expressing the music that ordinary maps don't show!

### <a id="songs"></a>Songs

Each contest provides beatmappers a choice between 5 songs from one of osu!'s Featured Artists.

> This month's artist is **MYLK**, and the following songs are available for viable for contest submissions:

Reviewer 4: ```suggestion
This month's artist is **MYLK**, and the following songs are available for contest submissions:
```

<audio controls>
    <source src="https://assets.ppy.sh/artists/62/previews/1759.mp3" type="audio/mpeg">
</audio>

#### Prism (100 BPM)

Beatmap template download: [Full version](https://assets.ppy.sh/artists/62/osz/MYLK%20-%20Prism.osz) (4:00) / [Short version](https://assets.ppy.sh/contests/84/files/MYLK_-_Prism.osz) (2:20)

<audio controls>
    <source src="https://assets.ppy.sh/artists/62/previews/1754.mp3" type="audio/mpeg">
</audio>

#### Daydreamer (170 BPM)

Beatmap template download: [Full version](https://assets.ppy.sh/artists/62/osz/MYLK%20-%20Daydreamer.osz) (4:10) / [Short version](https://assets.ppy.sh/contests/84/files/MYLK_-_Daydreamer.osz) (2:40)

<audio controls>
    <source src="https://assets.ppy.sh/artists/62/previews/1748.mp3" type="audio/mpeg">
</audio>

#### Let You Go (140 BPM)

Beatmap template download: [Full version](https://assets.ppy.sh/artists/62/osz/MYLK%20-%20Let%20You%20Go.osz) (3:00)

<audio controls>
    <source src="https://assets.ppy.sh/artists/62/previews/1752.mp3" type="audio/mpeg">
</audio>

> All beatmap templates above include shortened versions of songs for the sake of better beatmapping.

Reviewer 2: ```suggestion
All beatmap templates above include shortened versions of songs — mappers are free to map whichever one they prefer!
```

would avoid implying the short versions magically lead to better beatmaps

## <a id="screening"></a>Screening

While most beatmapping contests are judged solely by a panel of 3-5 judges, that isn't the case for the Monthly Beatmapping Contest.

We've selected **10** experienced beatmappers to provide their input and decide which 5 submissions proceed to community voting.

The week-long process of narrowing down submissions involves each member of the screening crew providing their top 3 entries in order of preference. The highest quality entry receives a value of 3, the next highest a value 2, and the final entry a value of 1. The top 5 are determined by the cumulative value of each entry.

> For transparency's sake, the screening users this month are [ScubDomino](https://osu.ppy.sh/users/8972308), [Noffy](https://osu.ppy.sh/users/1541323), [Zelq](https://osu.ppy.sh/users/8953955), [Uberzolik](https://osu.ppy.sh/users/1314547), [Venix](https://osu.ppy.sh/users/5999631), [Smokeman](https://osu.ppy.sh/users/2140676), [Sing](https://osu.ppy.sh/users/3795679), [Cris-](https://osu.ppy.sh/users/6175280), [Realazy](https://osu.ppy.sh/users/918297), and [Nuvolina](https://osu.ppy.sh/users/10974170). In case any of them drop out, back-up screening users are [Myxo](https://osu.ppy.sh/users/2202645), [Cheri](https://osu.ppy.sh/users/5226970), and [Sinnoh](https://osu.ppy.sh/users/4236057). All of these users will be reviewing the submissions without knowledge of who their creators are, and each of their preferred submissions will be viewable when the top 5 entries are announced.

Reviewer: would make sense to alphabetize the names at it looks quite random at the moment

> For transparency's sake, the screening users this month are [ScubDomino](https://osu.ppy.sh/users/8972308), [Noffy](https://osu.ppy.sh/users/1541323), [Zelq](https://osu.ppy.sh/users/8953955), [Uberzolik](https://osu.ppy.sh/users/1314547), [Venix](https://osu.ppy.sh/users/5999631), [Smokeman](https://osu.ppy.sh/users/2140676), [Sing](https://osu.ppy.sh/users/3795679), [Cris-](https://osu.ppy.sh/users/6175280), [Realazy](https://osu.ppy.sh/users/918297), and [Nuvolina](https://osu.ppy.sh/users/10974170). In case any of them drop out, back-up screening users are [Myxo](https://osu.ppy.sh/users/2202645), [Cheri](https://osu.ppy.sh/users/5226970), and [Sinnoh](https://osu.ppy.sh/users/4236057). All of these users will be reviewing the submissions without knowledge of who their creators are, and each of their preferred submissions will be viewable when the top 5 entries are announced.

Reviewer 5: ```suggestion
For transparency's sake, the screening users this month are  [Cris-](https://osu.ppy.sh/users/6175280), [Noffy](https://osu.ppy.sh/users/1541323),  [Nuvolina](https://osu.ppy.sh/users/10974170), [Realazy](https://osu.ppy.sh/users/918297), [ScubDomino](https://osu.ppy.sh/users/8972308), [Sing](https://osu.ppy.sh/users/3795679), [Smokeman](https://osu.ppy.sh/users/2140676), [Uberzolik](https://osu.ppy.sh/users/1314547), [Venix](https://osu.ppy.sh/users/5999631), and [Zelq](https://osu.ppy.sh/users/8953955). 

In case any of them drop out, back-up screening users are [Cheri](https://osu.ppy.sh/users/5226970), [Myxo](https://osu.ppy.sh/users/2202645), and [Sinnoh](https://osu.ppy.sh/users/4236057). 
All of these users will be reviewing the submissions without knowledge of who the creators are, and each of their preferred submissions will be view-able when the top 5 entries are announced.
```

Reviewer 5: Placed into alphabetical order

The voting page for this contest will be accessible upon the announcement of next month's contest.

## <a id="rewards"></a>Rewards

Similar to the rewards structure of the [Mappers' Guild](https://mappersguild.com), rewards for the Monthly Beatmapping Contest will be based on cumulative points earnings:
- **Community vote winner:** 2 points
- **Community vote runner-up:** 1 point
- **Screening crew's #1 pick:** 1 point

Each user's MBC points will be listed at the end of the news posts announcing a contest's winners.

MBC points reward users at certain thresholds:
- **Profile badge:** badges are attainable at 3, 6, and 10 points each. The badges are tiered, meaning each new badge will replace the previous one (and they get progressively shinier!).
- **"Elite Mapper" forum title**: attainable at 10 points.
- **osu!supporter:** each point earned provides a user with a month of supporter for themself or someone of their choosing.
- **Mappers' Guild participation:** participants who are in the Mappers' Guild will receive small earnings within the guild for submitting a valid entry (5), screening (1), and voting (1). These values are unrelated to the contest's points.

## Rules

> So how do you participate? Submissions must...

Reviewer 3: it's already a list i dont see the point of ellipses on every line

> - **...follow the month's [beatmapping limitation](#limitation): _all slider shapes must be straight lines (2 anchors)_.**

Reviewer: i think ``(2 anchors)`` might sound quite misleading to some people (wrongly assuming they could use straight slider + 2 red anchors), might be better going with something like ``(no extra sliderpoints)`` or so i guess?

- **...be in `.osz` format.** This allows your entry to use custom hitsound files and a background of your choice, however storyboarded visuals will be removed from all submissions.
- **...be a single difficulty for one of the month's [selected songs](#songs).**
- **...be in the _osu!_ game mode.** Other game modes will be considered after trialing this contest with osu! only.
- **...not be uploaded to the osu! website through the Beatmap Submission System (BSS).** Creators of each entry are anonymous during the screening and voting processes.
- **...abide by the Ranking Criteria.** While we can be forgiving for mistakes, submissions that could not be ranked without major changes will be discarded.

Any entries breaking these rules will not be viable for the contest.

---

> Head over to the [contest listing](https://osu.ppy.sh/community/contests/84) when you're ready to publish your entry. There's **3 weeks** until the submission form closes (2020-01-26), so launch the editor and get started as soon as you're ready.

Reviewer 2: ```suggestion
Head over to the [contest listing](https://osu.ppy.sh/community/contests/84) when you're ready to submit your entry. There's **3 weeks** until the submission form closes (2020-01-26), so launch the editor and get started as soon as you're ready.
```
they don't get published straight away right, so better to say submit.
